maestro/demos/workflows/cbom.ai/remediation/README.md

```diff
@@ -0,0 +1,156 @@
+# QSC Remediation
+
+The demo consists of three agents:
+
+| Agent | Purpose | In | Out | Tools | Notes |
+| -- | -- | -- | -- | -- | -- |
+|Finder|Parse CBOM to look for specific QSC issues to fix|CBOM|JSON findings|-|The mastercard CBOM from sonarqube scan is used|
+|Fixer|Fix code to make it safer|JSON findings|git email patch|fixer||
+|Patcher|Fix code to make it safer|git email patch|URL to PR|patcher||
+
+The Agents will together take a CBOM scan of a github repository, and by the end will have raised a pull request against
+the repo with a suggested fix.
+
+## Prereqs
+
+* Python 3.13
+* `pip install -r requirements.txt -U`
+* Agents are written to use the *BAM* runtime with *meta-llama/llama-3-1-70b-instruct* (see example config below)
+
+## Executor image
+
+This demo requires the following tools to be available within the executor environment:
+
+* git
+* gh (GitHub cli)
+
+### Building the image (optional)
+
+The standard executor image does not contain tools like 'gh' and 'git'
+
+Update the executor/Dockerfile to install additional packages
+
+```patch
+diff --git a/executor/Dockerfile b/executor/Dockerfile
+index 4f45222..5f3265b 100644
+--- a/executor/Dockerfile
++++ b/executor/Dockerfile
+@@ -42,8 +42,6 @@ RUN apk add --no-cache gcc musl-dev linux-headers make g++ clang-dev python3 pyt
+
+ FROM docker.io/alpine:${ALPINE_VERSION} AS runtime
+ RUN apk add --no-cache --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing \
+-    git \
+-    github-cli \
+     bash \
+     coreutils \
+     ffmpeg \
+```
+
+Then build. In my case I tag as follows since I push to quay.io
+
+```bash
+podman build -t quay.io/planetf1/bee-code-executor:20241117a executor
+podman push quay.io/planetf1/bee-code-executor:20241117a
+```
+
+### Using the executor image (mandatory)
+
+Update `bee-code-interpreter.yaml` in bee-stack to use the image you built above (or my prebuilt one)
+
+Look for the variable **APP_EXECUTOR_IMAGE** and update the value to the correct value for your newly created image. For example in the above build
+it would be updated to `quay.io/planetf1/bee-code-executor:20241117a`.
+
+## Running the demo
+
+### Command Line
+
+* Ensure the bee framework configuration is set appropriately. See an example later in this document.
+  * This should also be sourced into your current shell as the demo code requires it.
+* Start the bee agent framework as normal. Allow time (~1 minute) for the code executor to start
+* Launch a run-through of 3 sequenced agent with `./remediation_demo.py`
+
+### User Interface
+
+* Manually cleanup any old CBOM-AI Agents and tools using the bee UI
+* Set `BEE_KEEP=1`in the environment, and run the full demo through once. 
+* You can now use the bee UI to interact in the same way
+  * Some cut/paste of file contents may be needed
+
+## Debugging
+
+* Agents can be run individually within the bee framework via:
+  * `agents/run_finder.py`
+  * `agents/run_fixer.py`
```

Comment:
All agents should be maestro defined with yaml

Reply:
that's the plan. This is a copy of the old demo in it's mostly-working state